### ISSUE TITLE
Chest pain described as pressure

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10135]
**Preferred term label:**
Chest pain described as pressure

**Synonyms**

**Definition (free text, please give PubMed ID)**
Maybe https://uts.nlm.nih.gov/uts/umls/concept/C3647129 but it is hard to tell what that means

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
myocardial infarction

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000048
- label: Chest pain described as pressure
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: PMID:21250256
- def: Discomfort in the chest described as pressure, heaviness, or tightness.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10135
- synonyms:
